calc_skinit_lz_sums.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+FILE?=lz_header.bin
+
+extend_sha1 () {
+	HASH=`dd if=$FILE bs=1 count=$1 2>/dev/null | sha1sum | sed "s/ .*//" \
+		| sed "s/\([a-f0-9][a-f0-9]\)/\\\\\x\1/g"`
```

Comment:
These two sed's can be rather more succinctly done as:
`sed -r -e "s/ .*//g" -e "s/([a-f0-9]{2})/\\\x\1/g"`

Reply:
I've split this to a separate function and rewritten extend_sha* to take additional arguments. Hopefully this can be used to calculate expected PCR values at some point.